ticdc/src/main/java/io/tidb/bigdata/cdc/Misc.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2020 TiDB Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.tidb.bigdata.cdc;
+
+import io.tidb.bigdata.cdc.json.JsonNode;
+import io.tidb.bigdata.cdc.json.JsonParser;
+import io.tidb.bigdata.cdc.json.jackson.JacksonParserFactory;
```

Comment:
Unused import statement

Reply:
Oops, thanks for pointing out. It's fixed now.